File: explorer-backend/src/Models/API/TxResponse.cs
namespace ExplorerBackend.Models.API;

public class TxResponse
{
    public string? TxId { get; set; }
    public bool Confirmed { get; set; }
    public int BlockHeight { get; set; }
    public long Timestamp { get; set; }
    public int Version { get; set; }
    public int Size { get; set; }
    public int VSize { get; set; }
    public long Locktime { get; set; }
    public TransactionSimpleDecoded? Transaction { get; set; }
}

File: explorer-frontend/app/models/API/TxResponse.ts
import type { TransactionSimpleDecoded } from "@/models/API/BlockResponse";

export interface TxResponse {
  txId: string | null;
  confirmed: boolean;
  blockHeight: number;
  timestamp: number;
  version: number;
  size: number;
  vSize: number;
  locktime: number;
  transaction: TransactionSimpleDecoded | null;
}